uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.controller.js

```diff
@@ -597,6 +598,18 @@ define([
     function initWatchers () {
       $rootScope.$watch('cache.contact.arrSearch', collectContacts);
     }
+
+    /**
+     * Opens editor for activity type options editing
+     */
+    function openActivityTypeOptionsEditor () {
+      crmAngService.loadForm('/civicrm/admin/options/activity_type?reset=1')
+        .on('crmUnload', function () {
+          documentService.getOptions().then(function (options) {
```

Comment:
```js
documentService.getOptions()
  .then(function (options) {
    angular.extend($rootScope.cache, options);
  });
```
according to the styleguide https://compucorp.atlassian.net/wiki/spaces/PCHR/pages/121371880/AngularJS